src/CompilingMatcher.php

```diff
@@ -24,6 +24,9 @@ class CompilingMatcher
     private static $compiledCheckerCache = array();
     private static $enabled = null;
 
+    /**
+     * @psalm-var array<Constraint::OP_*, string>
```

Comment:
The const wildcards are wild! Very cool feature thanks @ondrejmirtes

Reply:
Cool! 😊 But we should use @phpstan- prefix here since this repo uses just PHPStan. The disadvantage of the @psalm- prefix is that if PHPStan finds something it cannot parse, it stays silent, but in case of @phpstan-, it will tell you that.

Reply:
> Cool! 😊 But we should use @phpstan- prefix here since this repo uses just PHPStan. The disadvantage of the @psalm- prefix is that if PHPStan finds something it cannot parse, it stays silent, but in case of @phpstan-, it will tell you that.

@ondrejmirtes thx for noticing. If not already part this should be mentioned in the docs. 

Never heared of this „limitation“ before. Good to know

Reply:
Can you send a PR that updates all `@psalm-` to `@phpstan-`? Thanks :)

Reply:
> Can you send a PR that updates all `@psalm-` to `@phpstan-`? Thanks :)

done in https://github.com/composer/semver/pull/99